nodes/mocha-testnet.md

```diff
@@ -80,7 +80,7 @@ Celestia network. The default port is 26657.
 - `celestia-rpc.f5nodes.com`
 - `celestia-testnet.brightlystake.com`
 - `rpc-celestia-mocha.architectnodes.com`
-- `rpc-celestia-mocha.testnet-pride.com`
+- `rpc-celestia-mocha.trusted-point.com`
```

Comment:
can you confirm you're replacing one of your own endpoints? @Hacker-web-Vi

Reply:
> can you confirm you're replacing one of your own endpoints? @Hacker-web-Vi

@jcstein confirming

Reply:
ty. :shipit: